cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -61,3 +61,27 @@
     not_if {File.exist? worktree_path}
   end
 end
+
+# Create sparse checkout of staging content directories on production-daemon
+# for the purpose of content seeding.
+if node.name == 'production-daemon'
+  worktree_path = File.join(home_path, 'staging')
```

Comment:
Maybe call this `staging-curriculum-content` to be more clear about what this other work tree is doing on the file system?

Reply:
Looks like configuring a worktree with a different path than the branch name is possible but would require a more complicated git command (and another round of testing with spinning up a new adhoc environment).